Library/Formula/teensy_loader_cli.rb

```diff
@@ -12,15 +11,34 @@ class TeensyLoaderCli < Formula
     sha256 "dcd10140babb4d2937ce376c89e9c24a2e8046d2cabdad2cfdbc2542afa14471" => :mavericks
   end
 
+  devel do
+    url "https://github.com/PaulStoffregen/teensy_loader_cli.git", :revision => "0cca2087afb54173ce03109cabb1e29658703fc8"
```

Comment:
Unfortunately no. Upstream put this app in maintenance mode. Last versioned release is outdated by few years though there are many new features in the repo. Especially support for new boards (Teensy/Arduino boards).